src/Sylius/Bundle/AdminBundle/Resources/config/routing/product_variant.yml

```
@@ -27,7 +25,7 @@ sylius_admin_product_variant_create:
                 method: createForProduct
                 arguments:
                     - expr:notFoundOnNull(service('sylius.repository.product').find($productId))
-            template: "@SyliusAdmin/Crud/create.html.twig"
+            template: "@SyliusAdmin/ProductVariant/Form/create.html.twig"
```

```suggestion
            template: "@SyliusAdmin/Shared/Crud/Form/create.html.twig"
```
because it does the same.